pkg/ExternalPackages/versions.props

```diff
@@ -2,19 +2,28 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
   <ItemGroup>
-    <ExternalPackage Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
+    <ExternalPackage Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
```

Comment:
Just here. The package is `runtime.win7-x64.runtime.native.system.data.sqlclient.sni`.

Same goes for clrcompression. Not sure how this was working with win7-amd64 as the RID unless it gets fixed somewhere in targets.